src/ontology/uberon-edit.obo

```diff
@@ -224180,6 +224180,16 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8480000
+name: iliac vein smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac vein." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001135 ! smooth muscle tissue
+intersection_of: part_of UBERON:0005610 ! iliac vein
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0005610 ! iliac vein
```

Comment:
This subclass axiom should be removed as it is already in the equivalent class axiom

Reply:
should I remove "relationship: part_of UBERON:0005610 ! iliac vein" and do the PR again, right?